utils/xtz_helpers.py

```diff
@@ -0,0 +1,178 @@
+import pysodium
+import eddsa_sign
+from decimal import Decimal
+from pytezos import pytezos
+from pyblake2 import blake2b
+from pytezos.operation.group import OperationGroup
+from base58 import b58encode_check, b58decode_check
+
+
+BIP_44_CONSTANT = 44
+XTZ_ASSET_NUM = 1729
+CHANGE = 0
+ADDR_INDEX = 0
+RPC = "https://rpc.tzbeta.net/"
+
+
+def tb(l):
+    return b''.join(map(lambda x: x.to_bytes(1, 'big'), l))
```

Comment:
Please use more verbose naming for function name and variables/parameters

Reply:
Done